### DISCUSSION TITLE
AGW clonning mac-address

### DISCUSSION THREAD

**Discussion by fabioccoelho** [magma/magma#11395] [Q&A] [answered]
Hi everyone,

I have an enviorement where AGW is configured as bridge mode, but when the customers log in into Mikrotik HotSpot, all customers mac-address are the same of uplink_br0 mac-address. 

I´ve already configure dp_router_enabled from True to False, but the customers get clonned mac-address  from AGW interface.

My enviorement like is:

Mikrotik as router ---> connected in AGW eth0 (SGi interface)
AGW eth1 --> connected in enodeb.

Into Magma Core I´ve configured Non-NAT mode and  DHCP_BROADCAST as IP Allocation Mode.

Is there any way to fix this behavior? I would like clients to receive the mac-address which is calculated by AGW. Can anyone help?

Edited: AGW and Magma Core are 1.6.0 version.

**Comment by edaspb** [comment, marked as answer]
Hi @fabioccoelho, I think it is impossible. We are using MACs which come from DHCP request for subscribers auth. They are unique.

**Reply by fabioccoelho** [reply]
Thanks :-)
